modules/installing-rhv-creating-custom-vm.adoc

```diff
@@ -0,0 +1,179 @@
+        storage_domain: <SSD_RAID_10> <6>
+        bootable: yes
+        timeout: 3600 <7>
+        upload_image_path: </custom/rhcos-44.81.202003110027-0-openstack.x86_64.qcow2> <8>
+        wait: yes
+----
+<1> For `<virtlab.example.com>`, specify the FQDN of your {rh-virtualization} Manager.
+<2> For `<username@profile>`, specify the admin username and profile.
+<3> For `<password>`, specify the admin password.
+<4> For `<rhcos_44-81_img-diskname>`, specify a disk name.
+<5> Specify `32GiB` or more of disk size.
+<6> For `<SSD_RAID_10>`, specify a data storage domain name.
+<7> Specify a timeout value in seconds that is appropriate for your environment.
+<8> For `</custom/rhcos-44.81.202003110027-0-openstack.x86_64.qcow2>`, specify the image path and filename.
++
+. On the Manager, run the Ansible playbook. For example:
++
+----
+$ ansible-playbook rhcos_image.yaml -vvv
+----
+
+. In the Administration Portal, go to the *Storage* -> *Disks* page and find the disk name you specified in the playbook. For example, `rhcos_44-81_img-diskname`.
+. When the status of the disk is *OK*, the playbook has finished uploading the RHCOS image to the storage domain.
+
+.Attach the virtual disk with the RHCOS image to the virtual machine
+
+. Go to the *Compute* -> *Virtual Machines* page and click btn:[New].
+. In the *New Virtual Machine* -> *General* window that opens, use *Cluster* to select the {rh-virtualization} cluster where you plan to create the cluster.
+. Leave *Template* unchanged, with *Blank* selected.
+. Set *Operating System* to *Red Hat Enterprise Linux CoreOS*.
+. Leave *Optimized for* unchanged, with *Desktop* selected.
+. For *Name*, specify the name of the virtual machine. For example, `vmname`.
+. For *Instance Images*, click btn:[Attach].
+. In the *Attach Virtual Disks* window that opens, find the disk you specified in the playbook. For example, `rhcos_44-81_img-diskname`.
+. Click the radio button for this disk.
+. Select the *OS* checkbox for this disk. This makes the disk bootable.
+. Click btn:[OK] to save and close the *Attach Virtual Disks* window.
+
+.Continue configuring the virtual machine and create it
+
+. In the *New Virtual Machine* -> *General* window, for each NIC under *Instantiate VM network interfaces by picking a vNIC profile*, select a vNIC profile.
+. Go to the *New Virtual Machine* -> *General* window.
++
+[IMPORTANT]
+====
+In the following steps, if you increase the values of *Memory* or *Total Virtual CPU* beyond the amounts shown, multiply these new values by the number nodes the installation program creates. By default, the installation program creates seven nodes. Verify that your {rh-virtualization} cluster has the memory and vCPU resources to meet these new values. For example, increasing *Total Virtual CPUs* from *4* to *6* means that your cluster requires 42 available vCPUs instead of 28. These vCPUs must be available exclusively for the cluster, and not required for non-cluster computing.
```

Comment:
This note and the following steps are confusing. I think that it makes more sense to make checking that your cluster has sufficient resources to make the VMs that you're instantiating a separate step after the customizations. Then, you can say that you need to multiply the values that you set by the number of machines that you plan to deploy, which is seven by default, and make sure that your RHV instance can support it.

s/number nodes/number of nodes that 

I'd s/meet these new values/that are required to support these values

s/cluster, and/cluster and